SF2Converter/converter/sf2_interface.cpp

```diff
@@ -16,7 +16,7 @@
 #include <assert.h>
 #include <sstream>
 #include <iostream>
-#include <filesystem>
```

Comment:
@Chordian I removed this include because it wouldn't compile in the Github build. It compiles fine now, also for me locally. Is this needed on Windows?

Reply:
On Windows it's needed for `setfill` and `setw` for `cout` to format numbers with a nicely formatted width.

Reply:
I put it in a conditional, only for windows

Reply:
Done.

Reply:
The branch still compiles fine one Windows after the changes.